app/views/home/_attributive_decision_form.html.haml

```diff
@@ -0,0 +1,5 @@
+= form_with method: :post, url: establishment_create_attributive_decisions_path(establishment) do |form|
+  .fr-checkbox-group.fr-mx-1w.fr-mb-1w
```

Comment:
on peut pas laisser les spacings par défaut ? et le faire remonter au DSFR ?

Reply:
Non en fait c'est pas la faute du DSFR, c'est moi qui ait un `fr-btns-group` autour de choses qui sont pas des boutons. Et cette classe a des marges négatives, compensées par les marges des boutons dedans (va savoir pourquoi).

J'utilise `fr-btns-group` pour faire en sorte que mes boutons prennent toute la largeur, mais en vrai je pourrais faire du css custom ça serait plus simple.

Reply:
Je viens faire ça mieux avec du css. J'ai aussi simplifié le code de "Manage spacing between buttons for home panel"